### PR TITLE
JBPM-6883 Stunner - can't move with bend points

### PR DESCRIPTION
**Pull request by tiagodolphine**
Now it is now possible to move bend points of a connector outside of the canvas area.
Note: I inserted a margin of 10px to avoid the bend point to be dropped exactly on the canvas bounds that was not easy to drag it back into the canvas.

@LuboTerifaj 
@wmedvede (adding you to help on the review since @romartin is on PTO)

related:
https://github.com/kiegroup/lienzo-core/pull/57

### CODE REVIEW

**Comment by tiagodolphine** [comment]
Jenkins execute full downstream build

**Comment by tiagodolphine** [comment]
hey @LuboTerifaj this issue is not related, see https://github.com/kiegroup/kie-wb-common/pull/2013 @hasys submitted a fix.

**Comment by hasys** [comment]
Hi @LuboTerifaj, @tiagodolphine,

fix already merged we can rerun build.

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by tiagodolphine** [comment]
@LuboTerifaj I think now it ok to test.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/components/drag/DragBoundsEnforcer.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import com.ait.lienzo.client.core.shape.wires.SelectionManager;
+import com.ait.lienzo.client.core.types.BoundingBox;
+import org.kie.workbench.common.stunner.core.client.shape.view.HasDragBounds;
+import org.kie.workbench.common.stunner.core.client.shape.view.ShapeView;
+import org.kie.workbench.common.stunner.core.graph.Graph;
+import org.kie.workbench.common.stunner.core.graph.Node;
+import org.kie.workbench.common.stunner.core.graph.content.Bounds;
+import org.kie.workbench.common.stunner.core.graph.content.definition.DefinitionSet;
+
+public class DragBoundsEnforcer {
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/components/drag/DragBoundsEnforcer.java]: nice job and and nice code :+1:

**Comment by tiagodolphine** [comment]
Hi @wmedvede can you help to review this one.
Thanks.

**Comment by tiagodolphine** [comment]
@wmedvede I think it is just missing the approval, since you already approved the lienzo PR.

**Comment by hasys** [comment]
Hi @manstis,

This PR is also ready to be merged, thank you!

**Comment by manstis** [comment]
@hasys Please let me know when the full downstream completes. Thanks.

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by tiagodolphine** [comment]
@manstis @hasys full downstream is ok now.

**Pull request merged**